package.json

```diff
@@ -0,0 +1,786 @@
+{
+  "name": "javascriptcapstone",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "webpack.config.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "watch": "webpack --config webpack.config.js --watch",
+    "start": "webpack serve --open"
```

Comment:
[Optional] Kindly include a test script in order to allow running the test. E.g `"test": "jest"`